runes.go

```diff
@@ -0,0 +1,31 @@
+package gana
+
+// SkipRunes skips the first `num` runes of a string by slicing (underlying array unaffected).
+func SkipRunes(num uint, what string) string {
+	if uint(len(what)) < num {
+		return ""
+	}
+	for i := range what {
```

Comment:
How does this work? So the `for` loop here when given strings goes through runes instead of bytes, therefore we can do a little bit of trickery of figuring out the rune index?

Reply:
`range` does, yes!